contracts/LiquidDemocracy.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.4;
+
+contract LiquidDemocracy {
+    ////////////////
+    // Data structures
+    ////////////////
+
+    // This is the type for a single option shown in a ballot
```

Comment:
Let's define better what a proposal is, is it a binary option? yes/no. I see only a votes variable, so it seems that you can vote for a proposal or not vote on it, and I hardly see the use case on that. I think it's better if we start simple, let's start with binary ones: we can add variables: positiveVoteCounts, negativeVoteCounts perhaps

Reply:
Agree with starting simple, let me run down the current data structures and see if we can think of something simpler. 

Right now there is:

- A`Proposal` struct with `name` and `voteCount` elements
- A `proposals` array that gets populated with `Proposal` structs in the constructor. The number of elements in this array is not limited to 2 right now, so technically it could be non-binary. But for testing simplicity I am actually creating binary ballot <`['yes', 'no'];`> in `LiquidDemocracy.test.js`. We should probably keep it binary indeed.

The naive/simple implementation of `vote()` simple increments the `voteCount` variable for the appropriate proposal option. I have a feeling this will be more clear to visualize once I finish writing the tests for `vote()` that I'm working on. Maybe for simplicity we can merge with data structures as they are right now but I definitely want to have this conversation on what's the most efficient and simple way to implement these data structures. 

What do you think?

Reply:
Ahaa I see, I misunderstood what a "proposal" is. Maybe i'm not too familiar with the terminology. So proposals are the options on which ones to vote: yes or no atm. Ok I think it's ok to leave it like this for now. Later on we can add more complexity to them. I think as you said it's better to have in mind that only yes/no proposals are accepted for now for simplicity

Reply:
Actually this thread is making me think we can do a better job with the naming, it probably is confusing enough the way it is right now. Will think about this moving forward and we can rename to a better option later.